src/bot/arena_bot.py

```diff
@@ -1,33 +1,55 @@
 from __future__ import annotations
 
+import aiohttp
 from glob import glob
 from pathlib import Path
 
-import aiohttp
 import discord
 from discord.ext.commands import Bot
 
+from config import COMMAND_PREFIX
+
 
 class ArenaBot(Bot):
-    def __init__(self, command_prefix, *args, **kwargs):
+    """
+    Subclass of `discord.ext.Bot`
+
+    This adds additional functionality:
+     * aiohttp session
+     * Singleton
+     * load_extensions
+    """
+    instance = None
+
+    # flake8: noqa: C901
```

Comment:
flake8 comment not necessary -> references a PEP standard which is not being raised by flake8 when linting.